apps/chameleon/components/ui/button.tsx

```diff
@@ -17,6 +17,8 @@ const buttonVariants = cva(
         secondary:
           'bg-secondary text-secondary-foreground hover:bg-secondary/80',
         ghost: 'hover:bg-accent hover:text-accent-foreground',
+        aware: 'bg-white text-[#191919] hover:bg-[#e5e5e5]',
```

Comment:
What is an aware button?

Reply:
the name aware is not a clear variant name